packages/react-core/src/components/AlertGroup/examples/AlertGroup.md

```diff
@@ -6,37 +6,68 @@ propComponents: ['Alert', 'AlertGroup', 'AlertActionCloseButton', 'AlertActionLi
 ---
 
 ## Examples
+
+### Variant examples
+
+Alert groups utilize the same variants that are available to [alert components](components/alert/). You can add multiple alert variants to each group.
+
 ### Static alert group
-These alerts appear on page load and are discoverable from within the normal page content flow, and will not be announced individually/explicitly to assistive technology.
+
+Static alert groups appear when the page loads. They are seen within the normal page content flow. They will not be announced individually or explicitly to assistive technology.
+
 ```ts file="./AlertGroupStatic.tsx"
 ```
 
-### Toast alert group
-Alerts asynchronously appended into dynamic AlertGroups with `isLiveRegion` will be announced to assistive technology at the moment the change happens, following the strategy used for aria-atomic, which defaults to false. This means only changes of type "addition" will be announced.
+### Toast alert groups
+
+Toast alerts appear in groups on top of content at the top right of the page. Click the buttons in the example below to simulate toast alerts on this webpage.
+
+Alerts that are asynchronously appended into dynamic `AlertGroups` with `isLiveRegion` will be announced to assistive technology at the moment the change happens, following the strategy used for `aria-atomic`, which defaults to `false`. This means only changes of type `addition` will be announced.
+
 ```ts file="./AlertGroupToast.tsx"
 ```
 
-### Toast alert group with overflow capture
-After a specified number of alerts displayed is reached, we will see an overflow message instead of new alerts. Alerts asynchronously appended into dynamic AlertGroups with `isLiveRegion` will be announced to assistive technology at the moment the change happens. When the overflow message appears or is updated in AlertGroups with `isLiveRegion`, the `View 1 more alert` text will be read, but the alert message will not be read. screen reader user or keyboard user will need a way to navigate to and reveal the hidden alerts before they disappear. Alternatively, there should be a place that notifications or alerts are collected to be viewed or read later. In this example we are showing a max of 4 alerts.
+### With overflow capture
+
+After a specified number of alerts displayed is reached, users will see an overflow message instead of new alerts. In the following example, an overflow message will appear when more than four alerts are queued.
+
 ```ts file="AlertGroupToastOverflowCapture.tsx" isBeta
 ```
 
-### Singular dynamic alert group
-This alert will appear in the page, most likely in response to a user action.
+### Dynamic alert groups 
+
+Dynamic alerts are generated after the page initially loads. Additional measures are required to ensure these alerts are accessible to screen readers.
```

Comment:
This could instead be worded more similar to the static alert group description, except noting that this type of alert group is used when alerts will dynamically update or appear.